example-app/app/books/components/book-search.ts

```diff
@@ -6,42 +6,39 @@ import { Component, Output, Input, EventEmitter } from '@angular/core';
 @Component({
   selector: 'bc-book-search',
   template: `
-    <md-card>
-      <md-card-title>Find a Book</md-card-title>
-      <md-card-content>
-        <md-input-container>
-          <input mdInput placeholder="Search for a book" [value]="query" (keyup)="search.emit($event.target.value)">
-        </md-input-container>
-        <md-spinner [class.show]="searching"></md-spinner>
-      </md-card-content>
-    </md-card>
+    <mat-card>
+      <mat-card-title>Find a Book</mat-card-title>
+      <mat-card-content>
+        <mat-input-container>
+          <input matInput placeholder="Search for a book" [value]="query" (keyup)="search.emit($event.target.value)">
+        </mat-input-container>
+        <mat-progress-spinner mode="indeterminate" [class.show]="searching"></mat-progress-spinner>
```

Comment:
Instead of using `mat-progress-spinner` you can use the `mat-spinner`. The `<mat-spinner>` component is an alias for `<mat-progress-spinner mode="indeterminate">`. You should also add the `diameter` and `strokeWidth` inputs. This will allow us to not have to use `!important` when setting the `width` and `height`. It should look something like:
```
<mat-spinner [class.show]="searching" [diameter]="30" [strokeWidth]="3"></mat-spinner>
```

Reply:
great i was struggling to set the correct values for this :)